lib/jackalope/tortoise_client.ex

```diff
@@ -83,10 +86,15 @@ defmodule Jackalope.TortoiseClient do
         {:stop, :missing_jackalope_process}
 
       %State{} = initial_state ->
-        {:ok, initial_state, {:continue, :spawn_connection}}
+        {:ok, initial_state, {:continue, :start_telemetry}}
```

Comment:
Why not call `send(self(), :capture_telemetry)` here instead of adding a `:continue` function?